apps/src/studio/studio.js

```diff
@@ -1184,6 +1184,20 @@ function handleActorCollisionsWithCollidableList (
             callHandler('whenGetAllItems');
           }
 
+          if (list == Studio.items) {
```

Comment:
use `===` - possibly move the code above into this block, so we don't need to check `list === Studio.items` twice.